monitoring/nodesstatus.go

```diff
@@ -172,16 +230,62 @@ type kubeNodeLister struct {
 	client corev1.CoreV1Interface
 }
 
-func formatCondition(condition v1.NodeCondition) string {
-	if condition.Message != "" {
-		return fmt.Sprintf("%v (%v)", condition.Reason, condition.Message)
-	}
-	return condition.Reason
-}
-
 const (
 	// NodeStatusCheckerID identifies the checker that detects whether a node is not ready
-	NodeStatusCheckerID = "nodestatus"
+	NodeStatusCheckerID = "node-status"
 	// NodesStatusCheckerID identifies the checker that validates node availability in a cluster
-	NodesStatusCheckerID = "nodesstatus"
+	NodesStatusCheckerID = "nodes-status"
+)
+
+var (
+	// DefaultNodeConditions is a list of all default Kubernetes node
+	// conditions.
+	DefaultNodeConditions = []v1.NodeConditionType{
+		v1.NodeOutOfDisk,
+		v1.NodeMemoryPressure,
+		v1.NodeDiskPressure,
+		v1.NodePIDPressure,
+		v1.NodeNetworkUnavailable,
+	}
+	// NodeProblemDetectorConditions is a list of node conditions
+	// commonly set by Node Problem Detector.
```

Comment:
`commonly` also has a touch of uncertainty - are they or aren't they set by NPD?
Also, since these are directly dependent on the version of planet - it should be at least mentioned to keep those in sync somehow.